src/unix/lwt_unix.ml

```diff
@@ -659,6 +659,114 @@ let write_string ch buf pos len =
   let buf = Bytes.unsafe_of_string buf in
   write ch buf pos len
 
+module IO_vectors =
+struct
+  type _bigarray =
+    (char, Bigarray.int8_unsigned_elt, Bigarray.c_layout) Bigarray.Array1.t
+
+  type _buffer =
+    | Bytes of Bytes.t
+    | Bigarray of _bigarray
```

Comment:
@avsm you're right here, I had forgotten that the leading `_` suppress dead code warnings. I removed the underscores in https://github.com/ocsigen/lwt/commit/e7a6755832f0888ac1e4dc4831158c0ba48de4b8.